### DISCUSSION TITLE
I have a problem at register by Education.

### DISCUSSION THREAD

**Discussion by portalninja2** [community/community#49631] [GitHub Education]
My Stundent Card is in Germany, and have two different pages, i have to upload both, but i cant only upload ONE image, what should i do?

**Comment by samus-aran** [comment]
Hi @portalninja2 

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. Ensure the important information is captured, including your current school, your name and date of study if you have it.
You can take a photo of both sides and combine them if you are permitted to upload your evidence.

Hope this helps.
